ReadMe.md

```diff
@@ -0,0 +1,30 @@
+# PRSDB - Infrastructure
+
```

Comment:
I would appreciate a context setting sentence here which: 

* Explains that this repo is just for the infrastructure 
* Explicitly point out that some architectural decisions affect infra for this service, but are documented elsewhere
* Links to the service repo